### ISSUE TITLE
Use uniform batch scripts

### ISSUE THREAD

**Issue by warpech** [StarcounterApps/Skyper#11]
Use uniform batch scripts as presented in https://starcounter.gitbooks.io/guidelines/content/batch-files.html

Discussion: https://github.com/Starcounter/Guidelines/issues/6

**Comment by warpech** [comment]
According to the discussion linked above, `make.bat` should be removed.

**Comment by warpech** [comment]
Some projects output the exe files to `./bin/Debug` instead of `./src/<appname>/bin/Debug`.

The latter form is expected (see https://starcounter.gitbooks.io/guidelines/content/project_structure.html)

If this solution does not output the exe files to the desired paths (both in msbuild and in VS), please fix that immediately too.

**Comment by Mackiovello** [comment]
Fixed with 	e46b2cc

**Comment by warpech** [comment]
After last change the "Debug" output path is good, but the "Release" output path should be changed, too: 

https://github.com/StarcounterApps/Skyper/blob/e46b2ccc3b8ac3ce255f71b4a8fa94f66b9c67fb/src/Skyper/Skyper.csproj#L35

Is: 

```
<OutputPath>..\..\bin\Release\</OutputPath>
```

Should be:

```
<OutputPath>bin\Release\</OutputPath>
```

**Comment by Mackiovello** [comment]
Fix according to new package guidelines

**Comment by Mackiovello** [comment]
Should finally be fixed with 	1b93924

**Comment by KmaTTi89** [comment]
Done in https://github.com/StarcounterApps/Skyper/pull/13
